.github/workflows/lint-unit.yml

```diff
@@ -77,7 +77,7 @@ jobs:
 
       - name: Prepare database
         run: |
-          bundle exec rails db:migrate
+          bundle exec rails db:drop db:create db:migrate
```

Comment:
Why not use `db:reset` ?

Reply:
[0d1f394](https://github.com/SUSE/rmt/pull/1198/commits/0d1f394f17e3ef58cdd636d288e46af87c040567)

https://github.com/SUSE/rmt/pull/1198#discussion_r1707105507

> It feels like this needs to be in a separate PR

without that the tests do not pass